### PR TITLE
Add bulk delete endpoints to the API

### PR DESCRIPTION
**Pull request by chrismytton**
This allows you to remove all the people, organizations, memberships and posts from an instance in one go. Alternatively you can just remove the records from an individual collection.

Fixes https://github.com/mysociety/popit/issues/464

### CODE REVIEW

File: test/rest.js [hunk 1]
```diff
@@ -875,4 +866,44 @@ describe("REST", function () {
 
   });
 
+  describe("bulk delete", function() {
+    beforeEach(fixture.loadFixtures);
+
+    it("works on individual collections", function(done) {
```
Comment (struan) [inline test/rest.js]: I'd put in a test here to make sure there's actually something in the collection.

Comment (chrismytton) [inline test/rest.js]: Ah good point, will do.

**Comment by struan** [comment]
One thing this doesn't do is remove the elastic search indexes so if you delete a collection and then do /search/persons it will still return things.

The deleting from mongo side looks fine though.

**Comment by chrismytton** [comment]
Oooh, good point. Shouldn't be too tricky to just drop the whole index for a given instance I don't suppose.

**Pull request merged**